model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -595,10 +621,6 @@ message StateResponse {
   // failed.
   string error = 2;
 
-  // (Optional) If this is specified, then the result of this state request
-  // can be cached using the supplied token.
-  bytes cache_token = 3;
```

Comment:
We could include the `cache_token` in the `StateRequest` from the SDK. I think this is not necessary if we assume that the Runner does not accept StateRequests from old connections with a SDK Harness.

Reply:
I don't think we need it as part of the request either since the state key uniquely describes which ProcessBundleDescriptor is being used as long as the runner doesn't reuse instruction ids.